artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,114 @@
-        - name: argoInstance2
-          url: https://argoInstance2.com
-          # dedicated username/password for this instance
-          username: ${ARGOCD_USERNAME_INSTANCE_2}
-          password: ${ARGOCD_PASSWORD_INSTANCE_2}
-----
+. For deployment details:
 
-* Define a token for each instance. This mapping has higher priority than both options mentioned previously.
 +
-[source]
-----
-argocd:
-  username: ${ARGOCD_USERNAME}
-  password: ${ARGOCD_PASSWORD}
-  appLocatorMethods:
-    - type: 'config'
-      instances:
-        - name: argoInstance1
-          url: https://argoInstance1.com
-          token: ${ARGOCD_AUTH_TOKEN} # Token to use to instance 1
-----
---
-
-=== Permissions in ArgoCD plugin
-
-Setting permissions for ArgoCD user account not only reduces the scope but also reduces the functionality of the backend. If you scope the permissions for read-only, then actions including creating, deleting, and resyncing the application will not be available. 
+* Select an appropriate card to view the deployment details (for example, commit message, author name, and deployment history) in a sidebar.
 
-The error handling in ArgoCD is designed to alert you when proper permissions are not in place.
-
-=== Self-signed certificates in ArgoCD plugin
-
-By default, the ArgoCD server generates a self-signed certificate. Once you install the ArgoCD plugin, the deployment of argocd-server can be patched to be insecure using the following command:
-
-[source,console]
-----
-kubectl patch deployment argocd-server --type "json" -p '[{"op":"add","path":"/spec/template/spec/containers/0/command/-","value":"--insecure"}]'
-----
-
-Also, you can use Helm to install the ArgoCD plugin and be insecure by default:
-
-[source,console]
-----
-helm upgrade --install argocd argo/argo-cd \
-  --version 3.33.5 \
-  --set 'server.extraArgs={--insecure}'
-----
-
-== For users
-
-=== Using ArgoCD plugin in {product}
-
-The ArgoCD plugin displays the current state of an application in Backstage Catalog.
-
-.Prerequisites
-
-* Your Backstage application is installed and running.
-* You have installed the ArgoCD plugin. For installation steps, see <<Installation>>.
-
-.Procedure
-
-. Open your Backstage application and go to the *Catalog* page.
-. In the *OVERVIEW* tab, you can see the ArgoCD overview card at the bottom.
 +
---
+image::rhdh-plugins-reference/sidebar.png[Sidebar]
 
-image::argo-cd-plugin-overview-card.png[ArgoCD overview card]
-
-The ArgoCD overview card displays application information including *NAME*, *INSTANCE*, *SYNC STATUS*, *HEALTH STATUS*, and *LAST SYNCED*.
---
-
-. To view the detailed application information, select an application from the ArgoCD overview card.
-+
---
-A pop-up containing detailed information about the application appears, along with a link to ArgoCD dashboard.
++ 
+* Select the link icon (image:rhdh-plugins-reference/link.svg[width=13px, height=13px, align=center]) to open the deployment details in Argo CD.
```

Comment:
Remove For deployment details

Format this as step 3: Optional: Click the [object] to view its deployment details, for example, commit message, author, name, and deployment history.
Format this is as a sub-step to step 3: Optional: Click the link icon to open the deployment details in Argo CD.

Use _click_ and _select_ in accordance with IBM stlye guideance throughout the doc

Reply:
> Remove For deployment details
Updated

> Format this as step 3: Optional: Click the [object] to view its deployment details, for example, commit message, author, name, and deployment history.
> Format this is as a sub-step to step 3: Optional: Click the link icon to open the deployment details in Argo CD.

Updated

> Use click and select in accordance with IBM stlye guideance throughout the doc
Raised question above